src/test/java/org/jabref/logic/importer/fetcher/AstrophysicsDataSystemTest.java

```diff
@@ -42,7 +42,7 @@ public void setUp() throws Exception {
         diezSliceTheoremEntry.setField(StandardField.EPRINT, "1812.04698");
         diezSliceTheoremEntry.setField(StandardField.JOURNAL, "arXiv e-prints");
         diezSliceTheoremEntry.setField(StandardField.KEYWORDS, "Mathematics - Differential Geometry, Mathematical Physics, 58B25, (58D19, 58B20, 22E99, 58A35)");
-        diezSliceTheoremEntry.setField(StandardField.MONTH, "Dec");
+        diezSliceTheoremEntry.setField(StandardField.MONTH, "#dec#");
```

Comment:
See https://github.com/JabRef/jabref/blob/fix-fetcher-tests/src/test/java/org/jabref/logic/importer/fetcher/AstrophysicsDataSystemTest.java#L46 for the current JabRef way to set months. - Please also note the addition to `toString()` in the PR - commenting on the # thing (https://github.com/JabRef/jabref/blob/5418f526664005f69e576ca8dee85ee8b3a720d8/src/main/java/org/jabref/model/entry/BibEntry.java#L613) -- even though I did not link our documentation ^^

Reply:
@tobiasdiez Finds it better - see https://github.com/JabRef/jabref/pull/6022#discussion_r386137064.